# Copyright (c) 2014 David Black et al. Licensed as AGPLv3.
# See https://gitorious.org/asdgasd/my-fdroiddata
Use consistent capitalization on button name.

diff --git a/res/layout/prefliste.xml b/res/layout/prefliste.xml
index c503322..2161049 100644
--- a/res/layout/prefliste.xml
+++ b/res/layout/prefliste.xml
@@ -2,7 +2,7 @@
 <LinearLayout
   xmlns:android="http://schemas.android.com/apk/res/android"
   android:orientation="vertical" android:layout_height="fill_parent" android:layout_width="fill_parent">
-<RelativeLayout android:id="@+id/LinearLayout01" android:layout_height="wrap_content" android:layout_width="fill_parent"><ImageView android:id="@+id/ImageView01" android:layout_width="wrap_content" android:layout_height="wrap_content" android:src="@drawable/icon" android:layout_alignParentLeft="true" android:layout_marginLeft="3dp"></ImageView><Button android:layout_centerVertical="true" android:layout_height="wrap_content" android:layout_width="wrap_content" android:text="Help" android:id="@+id/bt_help" android:layout_toLeftOf="@+id/bt_donate"></Button><Button android:layout_toRightOf="@+id/TextView01" android:layout_centerVertical="true" android:layout_width="wrap_content" android:id="@+id/bt_donate" android:layout_alignParentRight="true" android:layout_height="wrap_content" android:text="map pack"></Button>
+<RelativeLayout android:id="@+id/LinearLayout01" android:layout_height="wrap_content" android:layout_width="fill_parent"><ImageView android:id="@+id/ImageView01" android:layout_width="wrap_content" android:layout_height="wrap_content" android:src="@drawable/icon" android:layout_alignParentLeft="true" android:layout_marginLeft="3dp"></ImageView><Button android:layout_centerVertical="true" android:layout_height="wrap_content" android:layout_width="wrap_content" android:text="Help" android:id="@+id/bt_help" android:layout_toLeftOf="@+id/bt_donate"></Button><Button android:layout_toRightOf="@+id/TextView01" android:layout_centerVertical="true" android:layout_width="wrap_content" android:id="@+id/bt_donate" android:layout_alignParentRight="true" android:layout_height="wrap_content" android:text="Map pack"></Button>
 
 </RelativeLayout>
 <ListView android:layout_width="fill_parent" android:id="@+id/android:list" android:layout_height="wrap_content"></ListView>
